### PR TITLE
Fixes #601 - tree.html.twig throws NPE when currentSite is null

### LINKED ISSUE #601: tree.html.twig trying to access currentSite throws error when no sites are configured.
When using the sonata admin bundle to edit pages without a site configured you get an error:

```
Impossible to access an attribute ("name") on a null variable in SonataPageBundle:PageAdmin:tree.html.twig at line 52
```

lines 51-53 from tree.html.twig
```twig
<button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
    <strong class="text-info">{{ currentSite.name }}</strong> <span class="caret"></span>
</button>
```

however few lines down; line 58 is checking to make sure currentSite is not null.
```twig
{% if currentSite and site.id == currentSite.id %}
```

suggesting to check if current site is null on line 52 like the sample snippet below.
``` twig
<strong class="text-info">{{ currentSite ? currentSite.name : 'No sites configured' }}</strong> <span class="caret"></span>
```

I can make a pull request if this is appropriate.

### PR DESCRIPTION
**Pull request by StrikeForceZero**

### CODE REVIEW

File: Resources/translations/SonataPageBundle.en.xliff [hunk 1]
```diff
@@ -907,6 +907,10 @@
           <source>form.label_menu_template</source>
           <target>Menu template</target>
       </trans-unit>
+      <trans-unit id="226" resname="pages.current_site_null_label">
+          <source>pages.current_site_null_label</source>
+          <target>No sites configured</target>
```
Comment (OskarStark) [inline Resources/translations/SonataPageBundle.en.xliff]: german translation: "Keine Seiten konfiguriert" :smile:

File: Resources/views/PageAdmin/tree.html.twig [hunk 2]
```diff
@@ -48,8 +48,8 @@ file that was distributed with this source code.
                 <h1 class="box-title">
                     {{ 'pages.tree_site_label'|trans({}, 'SonataPageBundle') }}
                     <div class="btn-group">
-                        <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
-                            <strong class="text-info">{{ currentSite.name }}</strong> <span class="caret"></span>
+                        <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" {{ currentSite is null ? 'disabled' }}>
+                            <strong class="text-info">{{ currentSite is not null ? currentSite.name : 'pages.current_site_null_label'|trans({}, 'SonataPageBundle') }}</strong> <span class="caret"></span>
```
Comment (greg0ire) [inline Resources/views/PageAdmin/tree.html.twig]: These two lines are a bit too long.

File: Resources/views/PageAdmin/tree.html.twig [hunk 3]
```diff
@@ -48,8 +48,8 @@ file that was distributed with this source code.
                 <h1 class="box-title">
                     {{ 'pages.tree_site_label'|trans({}, 'SonataPageBundle') }}
                     <div class="btn-group">
-                        <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
-                            <strong class="text-info">{{ currentSite.name }}</strong> <span class="caret"></span>
+                        <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" {{ currentSite is null ? 'disabled' }}>
```
Comment (greg0ire) [inline Resources/views/PageAdmin/tree.html.twig]: Is `disabled` valid html or should this be changed to `disabled="disabled"`

**Comment by greg0ire** [comment]
Please rebase on the tip of master to get a green build

File: Resources/views/PageAdmin/tree.html.twig (hunk 3 above)
Comment (StrikeForceZero) [inline Resources/views/PageAdmin/tree.html.twig]: Per the [html5 spec](https://www.w3.org/TR/html5/forms.html#enabling-and-disabling-form-controls:-the-disabled-attribute) it is valid. Even per the [html4 spec](https://www.w3.org/TR/html4/interact/forms.html#adef-disabled). 

Only in [xhtml spec](https://www.w3.org/TR/xhtml1/dtds.html#dtdentry_xhtml1-strict.dtd_button) is `disabled="disabled"` required.

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**